### PR TITLE
Tweak result list, other small design touches

### PR DESCRIPTION
**Pull request by llorca**
- tweak result list visuals
- experiment with smaller URL when title exists
- make list scrolling full height by removing the expander
- add url icon to url results
- blue color for loading bar, we can revisit when we can display http vs https

### CODE REVIEW

File: src/components/resultList.js [hunk 1]
```diff
@@ -55,52 +52,115 @@ class ResultList extends Component {
...
+            source={require('../../images/search/url_result.png')}
+            style={styles.type, styles.typeIcon}
+          />
+        )
       case constants.RESULT_TYPE_HISTORY:
-        return(
+        return (
           <Image
             style={styles.type}
             source={{uri: 'http://' + result.domain + '/favicon.ico'}}
```
Comment (llorca) [inline src/components/resultList.js:65]: Need to check if this favicon exists somehow, right now we have some blank space with history URL that have no favicons

File: src/components/resultList.js [hunk 2]
```diff
@@ -55,52 +52,115 @@ class ResultList extends Component {
...
+          style={styles.title}
+        >
+          {title}
+        </Text>
+      );
+    } else if (urlExists) {
+      return (
+        <Text
+          numberOfLines={1}
+          style={[styles.url, styles.urlOnly]}
```
Comment (llorca) [inline src/components/resultList.js]: there's probably a better logic for that whole block... I was trying to have `styles.url` _and_ `styles.urlOnly` when there's no title. Used to tweak the font size of the URL whether there's a title or not

File: src/components/resultList.js (hunk 1 above)
Comment (spolu) [reply src/components/resultList.js:65]: I'll add a default image set to the result_url.png

Comment (spolu) [reply src/components/resultList.js:65]: Maybe we'll need another one for the default, but that works for now

**Comment by spolu** [comment]
LGTM

Simplified JSX logic
Set the favicon default

**Pull request merged**